nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/permissions/test/CommentFeature.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * (C) Copyright 2006-2019 Nuxeo SA (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Nour AL KOTOB
+ */
+
+package org.nuxeo.ecm.platform.comment.permissions.test;
+
+import org.junit.runner.RunWith;
+import org.nuxeo.ecm.core.test.annotations.Granularity;
+import org.nuxeo.ecm.core.test.annotations.RepositoryConfig;
+import org.nuxeo.ecm.platform.test.PlatformFeature;
+import org.nuxeo.runtime.test.runner.Deploy;
+import org.nuxeo.runtime.test.runner.Features;
+import org.nuxeo.runtime.test.runner.FeaturesRunner;
+import org.nuxeo.runtime.test.runner.LocalDeploy;
+import org.nuxeo.runtime.test.runner.SimpleFeature;
+
+@RunWith(FeaturesRunner.class)
+@Features(PlatformFeature.class)
+@RepositoryConfig(cleanup = Granularity.METHOD)
+@Deploy({ "org.nuxeo.ecm.relations.api", //
+        "org.nuxeo.ecm.relations", //
+        "org.nuxeo.ecm.relations.jena", //
+        "org.nuxeo.ecm.platform.comment.api", //
+        "org.nuxeo.ecm.platform.comment", //
+})
+@LocalDeploy("org.nuxeo.ecm.platform.comment.tests:OSGI-INF/comment-jena-contrib.xml")
+public class CommentFeature extends SimpleFeature {
```

Comment:
As this feature is only used for the old deprecated Comment service, I would:
- rename it to something more meaningful such as `CommentRelationFeature` maybe?
- deprecate it directly so that it will be removed with the deprecated service.
```
@deprecated since 10.10-HF12
```